<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>Workflow_name</th>
      <th>Annotation</th>
      <th>Workflow_file</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>60</th>
      <td>RNASeq DE (5 Groups, 3 Replicates_2 Factors) (READ ANNOTATION)</td>
      <td>RNA Sequencing Diff Expression Analysis of MOUSE samples using HISAT2, featureCounts, and multiple DE outputs (EdgeR, limmavoom). -PLEASE FEEL FREE TO CHANGE ALL NAMES OF INPUTS/OUTPUT STEPS; THESE ARE REFLECTIVE OF MY STUDY DESIGN. -If using different organism, please be sure to have the correct annotation file AND edit the alignment/count steps to the correct reference genome selection. -The first 6 samples are paired-end reads; if your sequence files are not paired-end, feel free to edit the HISAT2 step for these and remove the additional input files. -Alternatively, if you have more samples as paired-end reads, you will need to add additional inputs to the workflow and edit the respective HISAT2 steps to accept 2 fasta/q files to map your paired-end samples. -You will likely need to edit the Factors, groups, and contrast comparisons for your differential analysis steps (EdgeR, limmavoom, DESeq2). Be sure to input your factors, groups, and contrast instructions specific to your analysis/design. -All steps are setup to perform as UNSTRANDED; if you want to align and count as STRANDED, then you will need to edit all of the HISAT2 & featureCounts steps to reflect this.</td>
      <td><a href="https://drive.google.com/file/d/1oKfHxc1YiIF8TisSD-gOoha_ERcf6wkt/view?usp=sharing" target="_blank">https://drive.google.com/file/d/1oKfHxc1YiIF8TisSD-gOoha_ERcf6wkt/view?usp=sharing</a></td>
    </tr>
    <tr>
      <th>100</th>
      <td>Generic variation analysis on WGS PE data</td>
      <td>This workflows performs paired end read mapping with bwa-mem followed by sensitive variant calling across a wide range of AFs with lofreq</td>
      <td><a href="https://drive.google.com/file/d/18bwUoxBGPIA0kpjqdO5d_GM57CX-WcBX/view?usp=sharing" target="_blank">https://drive.google.com/file/d/18bwUoxBGPIA0kpjqdO5d_GM57CX-WcBX/view?usp=sharing</a></td>
    </tr>
    <tr>
      <th>40</th>
      <td>Rna-seq Proyect</td>
      <td>Clavulanic acid is a Î²-lactamase inhibitor, and is produced industrially by fermentative processes, using the bacterium Streptomyces clavuligerus. The efficiency of CA production is associated with media composition, culture conditions and the physiological and genetic characteristics of the strain. However, much of the molecular pathways governing AC regulation in S. clavuligerus remain unknown, and despite numerous studies, the regulatory mechanisms involved in the AC biosynthesis pathway have not yet been fully identified. In this work, a transcriptomic analysis was performed using RNA-seq data from S. clavuligerus strain ATCC 27064 which was grown in two culture media. The first was a complex, soy-based medium which was rich in nutrients. The second medium was a chemically defined, nutrient-poor medium. In total 1505 genes were found to be differentially expressed, 587 genes were expressed under the favourable conditions of soybean cultivation and 918 under unfavourable cultivation conditions. In addition, the string tool, available in cytoscape, was used to construct an interaction network with the data obtained in the differential expression analysis, using a confidentiality score of 0.7. An enrichment analysis was performed using string to identify which of these genes were involved in the production of antibiotics and other synthesis processes of interest, such as arginine production. We used the cluster Maker 2 tool and found a group of differentially expressed genes that contribute to the production of alanine which is an important precursor for the production of clavulanic acid. Transcriptome analysis in S. clavuligerus contributes to the identification of transcript abundance during cell development and environmental perturbations, which helps us to understand how the microorganism behaves under various conditions and thus contribute to the understanding of how clavulanic acid production functions.</td>
      <td><a href="https://drive.google.com/file/d/1N4XROLoKU945bgPoL6i7s-1T9KSi2Msm/view?usp=sharing" target="_blank">https://drive.google.com/file/d/1N4XROLoKU945bgPoL6i7s-1T9KSi2Msm/view?usp=sharing</a></td>
    </tr>
    <tr>
      <th>97</th>
      <td>RNA-seq preprocessing</td>
      <td>Preprocessing of RNA-seq data by quality trimming of read ends and Illumina adapter removal. Inputs: fastq files. Outputs: fastq files and html files with quality controls.</td>
      <td><a href="https://drive.google.com/file/d/1_A8ijabqPWnuQmkod8GwDcmgUGlagG3Y/view?usp=sharing" target="_blank">https://drive.google.com/file/d/1_A8ijabqPWnuQmkod8GwDcmgUGlagG3Y/view?usp=sharing</a></td>
    </tr>
    <tr>
      <th>33</th>
      <td>COVID-19: variation analysis on WGS SE (release v0.1.2)</td>
      <td>This workflows performs single end read mapping with bowtie2 followed by sensitive variant calling across a wide range of AFs with lofreq</td>
      <td><a href="https://drive.google.com/file/d/10wy2CV6q8mxLf_eZCAAV4Gh-jc7eHA-P/view?usp=sharing" target="_blank">https://drive.google.com/file/d/10wy2CV6q8mxLf_eZCAAV4Gh-jc7eHA-P/view?usp=sharing</a></td>
    </tr>
  </tbody>
</table>